students/mmancini/lesson09/donor_models.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+
+###################################
+
+
+class Donor():
+    def __init__(self, full_name):
+        self.name = full_name
+        self.lst_donations = []
+
+    def __init__(self, full_name, in_donations):
+        self.name = full_name
+        self.lst_donations = in_donations
+
+
+class DonorCollection():
+    def __init__(self, donors=None):
+        if donors is None:
+            donors = {}
+        self.dict_donors = donors
+
+    def add_donor(self, in_donor):
+        self.dict_donors[in_donor.name] = in_donor.lst_donations
```

Comment:
this class should hold reference to Donor objects - however does not appear to be the case in this class.